frontend/src/components/Town/interactables/Music/BrowseSongs.tsx

```diff
@@ -0,0 +1,336 @@
+import { Container } from '@chakra-ui/react';
+import React, { useEffect, useState } from 'react';
+import { InteractableID } from '../../../../types/CoveyTownSocket';
+import useTownController from '../../../../hooks/useTownController';
+import { Song } from './MusicArea';
+
+export default function MusicArea({
+  setRoute,
+  setCurrSong,
+}: {
+  interactableID: InteractableID;
+  route: string;
+  setRoute: React.Dispatch<React.SetStateAction<string>>;
+  currSong: Song | null;
+  setCurrSong: React.Dispatch<React.SetStateAction<Song | null>>;
+}): JSX.Element {
+  const [songs, setSongs] = useState<Song[]>([]);
+  const coveyTownController = useTownController();
+
+  /* TODO: At load this should fetch info from the backend, and load setSongs for now it's just a temporary array of fake data */
+  useEffect(() => {
+    /* TODO: Make database call to fetch the songs and load it into setSongs instead of the fakeSongs useEffect below*/
+  }, []);
+
+  /* TODO: Delete this, this just loads fake data into songs, used for styling purposes */
```

Comment:
will be resolved by [Music-Storage](https://github.com/M3CHR0M4NC3R/covey.town/tree/Music-Storage)
These fake songs are going into their own files and will be read in